src/main/java/org/fundacion/jala/converter/core/parameter/AudioParameter.java

```diff
@@ -16,7 +16,7 @@
 public class AudioParameter extends Parameter {
     private String format;
     private String bitrate;
-    private String frecuency;
+    private String frequency;
```

Comment:
K20_HERTZ have to be String or can be int

Reply:
It has to be a String :)